packages/cloud-cognitive/src/components/CreateTearsheet/CreateTearsheet.test.js

```diff
@@ -153,6 +155,45 @@ describe(CreateTearsheet.displayName, () => {
     expect(ref.current).not.toBeNull();
   });
 
+  it('should render the tearsheet on the specified initialStep prop provided', () => {
+    const { container } = renderCreateTearsheet({
+      ...defaultProps,
+      initialStep: 2,
+    });
+    const createTearsheetSteps = container.querySelector(
+      `.${createTearsheetBlockClass}__content .${carbon.prefix}--form`
+    ).children;
+    expect(
+      createTearsheetSteps[1].classList.contains(
+        `.${createTearsheetBlockClass}__step--visible-section`
+      )
+    );
+  });
+
+  it('renders the first step if an invalid initialStep value is provided', () => {
+    const warn = jest.spyOn(console, 'warn').mockImplementation(jest.fn());
+    const { container } = renderCreateTearsheet({
+      ...defaultProps,
+      // Starting on 0 step is invalid since the steps start with a value of 1
+      // This will cause a console warning
+      initialStep: 0,
+    });
+    const createTearsheetSteps = container.querySelector(
+      `.${createTearsheetBlockClass}__content .${carbon.prefix}--form`
+    ).children;
+    expect(
+      createTearsheetSteps[0].classList.contains(
+        `.${createTearsheetBlockClass}__step--visible-section`
+      )
+    );
+    expect(warn).toBeCalledWith(
+      `${componentName}: An invalid \`initialStep\` prop was supplied. The \`initialStep\` prop should be a number that is greater than 0 or less than or equal to the number of steps your ${componentName} has.`
```

Comment:
The component name is attached to the component render function as `displayName`, so we can use that to build the expected warning message.

```suggestion
      `${CreateTearsheet.displayName}: An invalid \`initialStep\` prop was supplied. The \`initialStep\` prop should be a number that is greater than 0 or less than or equal to the number of steps your ${CreateTearsheet.displayName} has.`
```